spec/author_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../classes/author'
+
```

Comment:
- [ ] Spec test for _Item_ class is missing! Kindly ensure to add a spec test for the _Item_ class which should include:
-  `can_be_archived?()` method with the following test conditions:
    * should return true if published_date is older than 10 years
    * otherwise, it should return false
- `move_to_archive()` method with the following test conditions:
    * should reuse can_be_archived?() method
    * should change the archived property to true if the result of the can_be_archived?() method is true
    * should do nothing if the result of the can_be_archived?() method is false

Reply:
Hi, @rloterh I think can_be_archived? test for item class can't be possible, because it is private in that class, though it is possible for the children class.

Thanks

Reply:
sorry, seems It shouldn't be private afterall